examples/error_handling.rs

```diff
@@ -0,0 +1,25 @@
+use tide::{After, Request, Response, Result, StatusCode};
+
+#[async_std::main]
+async fn main() -> Result<()> {
+    tide::log::start();
+    let mut app = tide::new();
+
+    app.middleware(After(|mut res: Response| async move {
```

Comment:
should `After` take both `request` and `response`? For example I might want to read a request headers and modify the response as appropriate. One example is taking the client request header from request and setting it to response header.

Reply:
Unfortunately that's not possible currently with After. If you need access to the request and the response, you'll need to write a full middleware